### PR TITLE
Add Support for CLI through open5gs-dbctl in HSS

### PR DESCRIPTION
**Pull request by platini2**

### CODE REVIEW

**Comment by platini2** [comment]
Pull request adding cli tought hss, howto added to README.md

**Comment by herlesupreeth** [comment]
Thanks for the PR. I will review and merge it over the weekend

File: base/Dockerfile [hunk 1]
```diff
@@ -68,7 +68,7 @@ RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg -
 
 # Get open5gs code and install
 RUN git clone --recursive https://github.com/open5gs/open5gs && cd open5gs && \
-    git checkout 322719f3e729aafacf531e85552d7a977fff3e2a && meson build --prefix=`pwd`/install && \
+    git checkout dcdfc970cefa00958f3712d3f81893253d5aa5ee && meson build --prefix=`pwd`/install && \
```
Comment (herlesupreeth) [inline base/Dockerfile:71]: please remove this change. I always want to use latest commits not old commits. Also, once you revert this change can you please test whether open5gs-dbctl works as intended?

Comment (platini2) [reply base/Dockerfile]: Done, apologies i was using a base file from a previous commit. Anyways. Now it is fixed

**Comment by herlesupreeth** [comment]
perfect..thank once again
